components/dashboard/basic/index.tsx

```diff
@@ -127,24 +183,15 @@ const BasicDashboard: React.FC = () => {
         container
         justifyContent="space-between"
         columnSpacing={5}
-        maxWidth={maxWidth}
+        width={maxWidth}
         mb={5}
       >
         <Metrics />
         <WordTable />
       </Grid>
-      <Grid container columnSpacing={5} maxWidth={maxWidth}>
+      <Grid container columnSpacing={5} width={maxWidth}>
```

Comment:
changed this so that the dashboard doesn't change width while the data is loading